client/src/pages/Playground/Playground.tsx

```diff
@@ -0,0 +1,192 @@
+// hooks/dependencies
+import React, { useState, useContext, useEffect} from 'react';
+import './Playground.scss';
+
+// graphiql
+import { GraphiQL } from 'graphiql';
+import { createGraphiQLFetcher } from '@graphiql/toolkit'
+
+// styles
+import { Button, Collapse } from 'antd';
+import 'graphiql/graphiql.min.css';
+
+const CopyToClipboard = require('react-copy-to-clipboard')
+
+const { Panel } = Collapse;
+const buttonStyle = {
+  color: 'var(--text-content)',
+  backgroundColor: 'var(--background-light)',
+  border: 'none'
+};
+const defaultStyle = {
+  backgroundColor: 'var(--background-light)'
+}
+
+// queries
+const query1 = `
+{
+  drugs(names: ["DOVITINIB"]) {
+    nodes {
+      interactions {
+        gene {
+          name
+          conceptId
+          longName
+        }
+        interactionScore
+        interactionTypes {
+          type
+          directionality
+        }
+        interactionAttributes {
+          name
+          value
+        }
+        publications {
+          pmid
+        }
+        sources {
+          sourceDbName
+        }
+      }
+    }
+  }
+}
+`;
+
+const query2 = `
+{
+  genes(names: ["BRAF"]) {
+    nodes {
+      interactions {
+        drug {
+          name
+          conceptId
+        }
+        interactionScore
+        interactionTypes {
+          type
+          directionality
+        }
+        interactionAttributes {
+          name
+          value
+        }
+        publications {
+          pmid
+        }
+        sources {
+          sourceDbName
+        }
+      }
+    }
+  }
+}
+`;
+
+const query3 = `
+{
+  drugs(names: ["IMATINIB"]) {
+    nodes {
+      name
+      conceptId
+      approved
+      immunotherapy
+      antiNeoplastic
+      drugAttributes {
+        name
+        value
+      }
+      drugApprovalRatings {
+        rating
+        source {
+          sourceDbName
+          sourceTrustLevel {
+            level
+          }
+        }
+      }
+      drugApplications {
+        appNo
+      }
+    }
+  }
+}
+`;
+
+const query4 = `
+{
+  genes(names: ["BRAF"]) {
+    nodes {
+      longName
+      conceptId
+      geneCategoriesWithSources {
+        name
+        sourceNames
+      }
+    }
+  }
+}
+`
+
+const fetcher = createGraphiQLFetcher({ url: 'http://localhost:3000/api/graphql'})

```

Comment:
@acoffman @jsstevenson this works locally for me, but for deployment purposes, what should this point to? (I think there is likely an environment variable setting to use here but I do not know the best implementation)

Reply:
I would just import and use the `API_URL` const that's already available and pulled in from the env, that should work for you!